generated/global-autoconfig-properties/src/main/java/com/google/cloud/spring/global/GlobalProperties.java

```diff
@@ -0,0 +1,19 @@
+package com.google.cloud.spring.global;
```

Comment:
Could you add a license header to this file (similar to the other files we are generating), and run `google-java-format` (see [repo's style guide](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/main/CONTRIBUTING.md#code-style)) to fix the formatting here? 

Just realized that `mvn verify` (through the integration tests workflow) runs checkstyle in this repo, so the check probably slipped away when we disabled presubmit ITs for codegen PRs.

Reply:
Fixed format and added header

Reply:
@zhumin8 you are right - I just saw that checkstyle is actually bound to the [validate](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/0489cf907df695379b9d1d4a41a0c9b3241ccfb0/pom.xml#L260) phase, which is covered by unit tests. I had the wrong understanding here earlier when trying to reason why the checks didn't catch this file.

In that case, at the time of this comment, this was probably not picked up because the global-autoconfig-properties wasn't bound to the generated-parent pom yet (which has now been updated).

Reply:
Checkstyle currently [passing](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/actions/runs/3577646038/jobs/6016883951#step:7:217)